notebooks/MAST/dwarf_galaxy_search/dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<p>Please change these below.</p><p> </p><blockquote>A vast majority of these satellites</blockquote><p>These satellites are classified into two major classes: Classical dwarf spheroidal and ultra-faint dwarf (UDF) galaxies.</p><ul><li>Please add a simple description (definition) of why each galaxy is.</li></ul><p> </p><blockquote>explote</blockquote><p>explore</p><p> </p><blockquote>We will be searching the <a href="https://outerspace.stsci.edu/display/PANSTARRS/" target="_blank">Pan-STARRS</a> (PS1) catalog for data on an already discovered dwarf galaxy.</blockquote><p>We will be searching the <a href="https://outerspace.stsci.edu/display/PANSTARRS/" target="_blank">Pan-STARRS</a> (PS1) catalog for data on an already discovered dwarf galaxy and then try to search if there are other MAST data associated with the galaxy.</p><p> </p><blockquote>we will choose another target,</blockquote><p>we will choose another target, a UFD galaxy</p><p> </p><blockquote>Lastly, we will search Hubble Space Telescope data to plot a spectral image of our first target.</blockquote><p>comment out this please.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>